defaults/defaults_unix.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```

Comment:
This branch is still on go1.16, so looks like these changes must be reverted 😞 

```
Error: File is not `gofmt`-ed with `-s` (gofmt)
Error: File is not `gofmt`-ed with `-s` (gofmt)
Error: File is not `gofmt`-ed with `-s` (gofmt)
```

Not sure which file this is complaining about though 😞 let me try running it locally

```
Warning: error-strings: error strings should not be capitalized or end with punctuation or a newline (revive)
Warning: error-strings: error strings should not be capitalized or end with punctuation or a newline (revive)
```